var/spack/repos/builtin/packages/py-dask/package.py

```diff
@@ -12,6 +12,7 @@ class PyDask(PythonPackage):
     homepage = "https://github.com/dask/dask/"
     url      = "https://pypi.io/packages/source/d/dask/dask-1.1.0.tar.gz"
 
+    version('1.2.2', sha256='5e7876bae2a01b355d1969b73aeafa23310febd8c353163910b73e93dc7e492c')
     version('1.1.2', sha256='93b355b9a9c9a3ddbb39fab99d5759aad5cfd346f4520b87788970e80cf97256')
     version('1.1.0', sha256='e76088e8931b326c05a92d2658e07b94a6852b42c13a7560505a8b2354871454')
     version('0.17.4', sha256='c111475a3d1f8cba41c8094e1fb1831c65015390dcef0308042a11a9606a2f6d')
```

Comment:
Things to add:
```python
      python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*",     
```
Also, `py-pytest-runner` is needed as a test dep. And some extras dependency versions have changed:
```python
extras_require = {                                                              
  'array': ['numpy >= 1.13.0', 'toolz >= 0.7.3'],                               
  'bag': ['cloudpickle >= 0.2.1', 'toolz >= 0.7.3', 'partd >= 0.3.8'],          
  'dataframe': ['numpy >= 1.13.0', 'pandas >= 0.21.0', 'toolz >= 0.7.3',        
                'partd >= 0.3.8', 'cloudpickle >= 0.2.1'],                      
  'distributed': ['distributed >= 1.22'],                                       
  'delayed': ['toolz >= 0.7.3'],                                                
}
```